examples/node_test/node_test.cpp

```diff
@@ -0,0 +1,162 @@
+//
+// Simple example to test out multiple nodes
```

Comment:
For tests I suggest to use integration_tests instead of examples.

Reply:
@julianoes Will do eventually, right now node_test is just my quick way of testing new code, I'll port it over before merge.